### DISCUSSION TITLE
Welcome to mimikatz Discussions!

### DISCUSSION THREAD

**Discussion by upenderadepu** [upenderadepu/mimikatz#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sphiinx1** [comment]
Ich habe verstanden dass man mit Mimikatz passwörter herrausfinden kann aber um dies zu können muss man bereits auf dem Sytem sein meine frage ist aber wie komme ich in das system gibt dafür tools?
